workflows/comfystream/sd15-multi-cnet-depthanything-face-api.json

```
@@ -260,7 +260,7 @@
   },
   "26": {
     "inputs": {
-      "image": "example.png",
+      "image": "harold.png",
```

Let's make harold more generic
```suggestion
      "image": "example.png",
```